lib/report_formatter/chart_common.rb

```diff
@@ -156,6 +156,7 @@ def build_performance_chart_pie(_maxcols, divider)
       return no_records_found_chart if series.empty?
 
       add_axis_category_text(categories)
+      series.zip categories { |ser, category| ser[:tooltip] = category }
```

Comment:
@martinpovolny because I need to have series in format like this:
``` [{:value=>926, :tooltip=>"PRDECM001"}, {:value=>436, :tooltip=>"PRDEVM001"}]
```